master/internal/db/postgres_tasks.go

```diff
@@ -136,6 +136,15 @@ DO UPDATE SET
 `, a)
 }
 
+// AddAllocationExitStatus adds the allocation exit status to the allocations table.
+func (db *PgDB) AddAllocationExitStatus(a *model.Allocation) error {
```

Comment:
make this method take a context, and don't have it be on ```*PgDB```, and make the ```db.sql.Exec``` use Bun instead

(its okay if in allocation.go we need to use context.TODO() to call it)

Reply:
Done!